firebase/package.json

```diff
@@ -21,9 +22,10 @@
     "build": "run-p --print-label build:*",
     "build:functions": "tsc -p tsconfig.functions.json",
     "functions:serve": "run-s --print-label build:functions functions:start-emulators watch",
-    "functions:fetch-env": "firebase functions:config:get > src/core/.runtimeconfig.json",
```

Comment:
The env file must be placed in the root instead in the `core` folder, otherwise firebase emulator can't locate it